src/ui/Quickview/QuickviewDocument.ts

```diff
@@ -45,7 +46,11 @@ export class QuickviewDocument extends Component {
      *
      * Default value is `0`, and the index returns the entire preview. Minimum value is `0`.
      */
-    maximumDocumentSize: ComponentOptions.buildNumberOption({ defaultValue: 0, min: 0 })
+    maximumDocumentSize: ComponentOptions.buildNumberOption({ defaultValue: 0, min: 0 }),
+    /**
+     * Specifies the title of the document.
+     */
+    title: ComponentOptions.buildStringOption()
```

Comment:
Is it preferable for me to add a `title` option like this or for me to use a `QuerySelector` higher up and add it to the `iframe`?

If this option is kept, what should be the description here? Should it be mentioned here that this option's sole purpose is passing a `title` to the `iframe` for accessibility tools to recognize?

@fbeaudoincoveo

Reply:
Would it be possible to infer the title from the indexed document we are trying to display?

Reducing the number of options and configuration a developer needs to do is usually the better path.

Moreover, we are logging a warning for what is currently an non-required option. This suggests that the option is, if not required, important to specify.

Reply:
Exactly as Sami said, this option is not necessary and can already be inferred from other properties ( either the index result title, or, as suggested in a following comment, the HTMLDocument title itself).

This should be removed.

Reply:
Good point, I removed the option and used the `title` in the received `IQueryResult` instead.